enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+    reason: NoEvaluationCondition
+    status: "False"
+    type: EvaluationConditionsDetected
+  - lastTransitionTime: "2024-04-09T22:43:21Z"
+    message: Canary route checks for the default ingress controller are successful
+    reason: CanaryChecksSucceeding
+    status: "True"
+    type: CanaryChecksSucceeding
+  domain: apps.misalunkeip16.devcluster.openshift.com
+  endpointPublishingStrategy:
+    loadBalancer:
+      dnsManagementPolicy: Managed
+      providerParameters:
+        aws:
+          classicLoadBalancer:
+            connectionIdleTimeout: 0s
+          type: Classic
+        type: AWS
+      scope: External
+    type: LoadBalancerService
+  observedGeneration: 1
+  selector: ingresscontroller.operator.openshift.io/deployment-ingresscontroller=default
+  tlsProfile:
+    ciphers:
+    - ECDHE-ECDSA-AES128-GCM-SHA256
+    - ECDHE-RSA-AES128-GCM-SHA256
+    - ECDHE-ECDSA-AES256-GCM-SHA384
+    - ECDHE-RSA-AES256-GCM-SHA384
+    - ECDHE-ECDSA-CHACHA20-POLY1305
+    - ECDHE-RSA-CHACHA20-POLY1305
+    - DHE-RSA-AES128-GCM-SHA256
+    - DHE-RSA-AES256-GCM-SHA384
+    - TLS_AES_128_GCM_SHA256
+    - TLS_AES_256_GCM_SHA384
+    - TLS_CHACHA20_POLY1305_SHA256
+    minTLSVersion: VersionTLS12
+```
+
+Add the `ingress.operator.openshift.io/auto-delete-load-balancer:` to the ingress controller.
+```sh
+% oc -n openshift-ingress-operator annotate ingresscontrollers/default ingress.operator.openshift.io/auto-delete-load-balancer=
+ingresscontroller.operator.openshift.io/default annotated
+```
+
+Add the following under spec:
+```yaml
+spec:
+  endpointPublishingStrategy:
+    loadBalancer:
+      dnsManagementPolicy: Managed
+      providerParameters:
+        aws:
+          networkLoadBalancer:
+            eipAllocations:
+              - eipalloc-0956fea34de4cb7ab
+              - eipalloc-0e9a3077a70de050a
+              - eipalloc-0b69fc4691f54cdd0
+              - eipalloc-01e6ba6cbba1a391b
+              - eipalloc-0e242df173f906112
+          type: NLB
+        type: AWS
+      scope: External
+    type: LoadBalancerService
+```
+
+Check if the router-default service was created with proper annotations.
+
+```yaml
+% oc get svc router-default -n openshift-ingress -o yaml         
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-eip-allocations: eipalloc-0956fea34de4cb7ab,eipalloc-0e9a3077a70de050a,eipalloc-0b69fc4691f54cdd0,eipalloc-01e6ba6cbba1a391b,eipalloc-0e242df173f906112
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-healthy-threshold: "2"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval: "10"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-timeout: "4"
+    service.beta.kubernetes.io/aws-load-balancer-healthcheck-unhealthy-threshold: "2"
+    service.beta.kubernetes.io/aws-load-balancer-type: nlb
+    traffic-policy.network.alpha.openshift.io/local-with-fallback: ""
+  creationTimestamp: "2024-04-10T08:04:41Z"
```

Comment:
We don't need to see anything after the annotations in the Service yaml.

Reply:
I just updated it.